src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+    private double area;
```

Comment:
actually you don't need this for figures, see comment in AreaCalculable, you can return area in method getArea()